backend/src/repositories/AccelerationRepository.ts

```diff
@@ -308,10 +308,10 @@ class AccelerationRepository {
         }
         const accelerationSummaries = accelerations.map(acc => ({
           ...acc,
-          pools: acc.pools.map(pool => pool.pool_unique_id),
+          pools: acc.pools,
         }))
         for (const acc of accelerations) {
-          if (blockTxs[acc.txid] && acc.pools.some(pool => pool.pool_unique_id === block.extras.pool.id)) {
+          if (blockTxs[acc.txid] && acc.pools.some(pool => pool === block.extras.pool.id)) {
```

Comment:
I suppose you could change that to an `.includes` instead since it's a simple array of number now.